telegraf/plugins/outputs/outputs_test.go

```diff
@@ -65,22 +122,45 @@ func TestTOML(t *testing.T) {
 				}: `[[outputs.file]]
   ## Files to write to, "stdout" is a specially handled file.
   files = ["stdout", "/tmp/out.txt"]
+
+  ## Use batch serialization format instead of line based delimiting.  The
+  ## batch format allows for the production of non line based output formats and
+  ## may more efficiently encode metric groups.
+  # use_batch_format = false
+
+  ## The file will be rotated after the time interval specified.  When set
+  ## to 0 no time based rotation is performed.
+  # rotation_interval = "0d"
+
+  ## The logfile will be rotated when it becomes larger than the specified
+  ## size.  When set to 0 no size based rotation is performed.
+  # rotation_max_size = "0MB"
+
+  ## Maximum number of rotated archives to keep, any older logs are deleted.
+  ## If set to -1, no archives are removed.
+  # rotation_max_archives = 5
+
+  ## Data format to output.
+  ## Each data format has its own unique set of configuration options, read
+  ## more about them here:
+  ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_OUTPUT.md
+  data_format = "influx"
 `,
 				&InfluxDBV2{
 					URLs: []string{
-						"http://192.168.1.10:8086",
-						"http://192.168.1.11:8086",
+						"http://192.168.1.10:9999",
```

Comment:
What is the significance of changing these to 9999? The default port for OSS is 8086 - does this show up anywhere but the tests?

Reply:
No significance. Just for the tests. I think the example in the comments of the Telegraf plugins use 9999 and that's where it came from.

Reply:
10-4, seems fine then!